faq.md

```diff
@@ -2,15 +2,15 @@
 
 ## Are there travel funds available?
 
-Not currently. We are pursuing some potential sources and will make announcements here are elsewhere if funding becomes available.
+Unfortunately, we were unable to secure travel funds for this event.
 
 ## Can I participate remotely?
 
 Yes! Our meeting rooms will be equipped for hybrid events. We will have at least two synchronous plenary sessions each day so remote participants will be able to present work to the whole group.
 
 ## Is there a reserved hotel block for this event?
 
-Yes! _Need to update this_
+Yes! We requested 20 rooms at the _______.
```

Comment:
Is the hotel still unknown?

Reply:
Yes. I asked Taysia for an update on it yesterday, so hopefully we hear back soon.